infrastructure/docker/services/api/etc/nginx/nginx.conf

```diff
@@ -27,6 +27,8 @@ http {
     server {
         listen 0.0.0.0:80;
         root /home/app/application/public;
+        fastcgi_buffers 16 16k;
```

Comment:
can be remove, added in previous PR https://github.com/maximelafarie/humand/pull/11/files